_includes/tutorials/aggregating-count/ksql/code/tutorial-steps/dev/outputs/persistent-query/output-0.log

```diff
@@ -1,4 +1,4 @@
-Message                   
----------------------------
- Table created and running 
----------------------------
+Message                                                                                                 
```

Comment:
I've noticed some of these output files are checked in, but get deleted and re-created with each run.  Should we start to not include these files under source control?   Ditto for the other output log files in this PR.

Reply:
Yeah, we opened #217 regarding this. Let's come back through later and remove these. Fine to leave for the moment.